templates/spiders/json_blob.tmpl

```diff
@@ -11,7 +12,6 @@ class $classname(JSONBlobSpider):
         # "brand_wikidata": "$brand_wikidata",
         # "operator": "$operator",
         # "operator_wikidata": "$operator_wikidata",
-        # "extras": Categories.SHOP_XYZ.value
     }
     start_urls = ["$url"]
     # no_refs = True
```

Comment:
I don't mind `no_refs` being listed but it is a very rare occasion we'd be using it, so may not be necessary to include in templates?